components/form/checkboxesWithParent/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@schibstedspain/sui-form-checkboxes-with-parent",
```

Comment:
if this is not a sui component the name should be `@schibstedspain/react-form-checkboxes-with-parent`

Reply:
Other components on this repository has same name convention. For example:
- "name": "@schibstedspain/sui-form-checkbox"
- "name": "@schibstedspain/sui-card-basic"

What to do?

Reply:
these components were previously sui and after moved to this repository. all the new components in this repo sould be `@schibstedspain/react-xxxxx` like:

https://github.com/SUI-Components/schibsted-spain-components/blob/master/components/cmp/modal/package.json#L2

Reply:
Package name updated, but we must take into account that all components on this repository has `sui-` prefix on their package names.